fhir_zulip_nlp/fhir_zulip_nlp.py

```diff
@@ -345,6 +304,41 @@ def query_categories(category_keywords: Dict[str, List[str]] = KEYWORDS) -> pd.D
     return df_raw
 
 
+# TODO: need to account for spelling variations w/in a new colum in the google sheet
+# todo: $validate-code: there are actually 2 different operations w/ this same name. might need to disambiguate
+def _get_keywords(
+    sheet_name='category_keywords', cache_path=CONFIG['keywords_cache_path'], use_cache_only=False
+) -> Dict[str, List[str]]:
+    """Get keywords from google sheets, else cache
+    sheet_name: The name of the specific sheet within a GoogleSheet workbook."""
+    # Load
+    if use_cache_only:
+        df: pd.DataFrame = pd.read_csv(cache_path)
+    else:
+        try:
+            # todo: Pass URI too
+            df: pd.DataFrame = get_sheets_data(sheet_name=sheet_name, env_dir=ENV_DIR)
+            df.to_csv(cache_path, index=False)
+        except:
```

Comment:
Just in case anything goes wrong with trying to fetch from google sheets, I added a blind `except` that will simply load the cached CSV file that we have.